CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
          should enable it themselves after assessing its startup impact.
 - Removed support for the previously deprecated `OE_API_VERSION=1` APIs.
 - Update MUSL libc to version 1.1.21.
+- Update mbedTLS to version 2.16.2.
```

Comment:
Shouldn't we create a new v0.7 changed section instead of editing the section for v0.6?

Reply:
Thanks, I messed up a merge with master, it should have replaced the 2.7.11 line in the Unreleased section. Fixed.